### ISSUE TITLE
error: could not find module 'Usabilla' for target 'i386-apple-ios-simulator'; found: x86_64-apple-ios-simulator, arm64, x86_64, armv7-apple-ios, arm64-apple-ios, arm, armv7

### ISSUE THREAD

**Issue by RikSchefferOberon** [usabilla/usabilla-u4a-ios-swift-sdk#211]
Starting at Usabilla 6.5.0 I'm receiving the following error during buildtime:
error: could not find module 'Usabilla' for target 'i386-apple-ios-simulator'; found: x86_64-apple-ios-simulator, arm64, x86_64, armv7-apple-ios, arm64-apple-ios, arm, armv7

Previously I was on 6.4.3, which contained the i386 targets.

I have installed Usabilla through cocoapods.

Does anyone have a solution other than reverting to 6.4.3?

**Comment by christinasund** [comment]
We had this issue as well, but updated the pod spec to set the `EXCLUDED_ARCHS` to `arm64 i386`. I am assuming this will need to be resolved at some point, but it works for now. Here is some info for you on how to give this a try in your Podfile: https://github.com/CocoaPods/CocoaPods/issues/10104#issuecomment-704862852

**Comment by eduakaszas** [comment]
Hi both, 

@christinasund thank you for the suggested solution! @RikSchefferOberon would you mind trying this out and let us know if it resolves the issue for you as well? If it doesn't, we'll investigate further. 

Kind regards, 
Édua

**Comment by RikSchefferOberon** [comment]
the suggestion did work for me. So adding the following to the podspec solved it:
`s.pod_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64'}`
`s.user_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64'}`

**Comment by lewie9021** [comment]
I'm a little confused why this issue has been closed?

In my React Native project using [usabilla-u4a-react-native](https://github.com/usabilla/usabilla-u4a-react-native) (`0.12.1`), I get the same error detailed in this issue.

The patch for me was similar to what was suggested by @RikSchefferOberon, only I had to also include `i386` which had to be placed in **your** `.podspec` (`node_modules/usabilla-react-native/usabilla-react-native.podspec`) file:
```
s.pod_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64 i386' }
s.user_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64 i386' }
```

From my limited knowledge, presumably a long term solution is to either apply the above change officially or continue to support the i386 architecture as it did previously?

**Comment by eduakaszas** [comment]
Hi @lewie9021, 

I indeed should not have closed the ticket, that was my mistake. Apologies for that!

Since we have excluded arm64 from our SDK for simulators, we have added the below to our `.podspec`file with the new SDK release:
```
s.pod_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64' }
s.user_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64' }
```

For React Native, we build for all architectures except for arm64 for simulators.

As for `i386`, we have run into some issues during the new SDK release, which we're looking into. We'll keep you posted about that!

Let me know if you have any questions. 

Kind regards, 
Édua
